frontend/pages/policies/ManagePoliciesPage/components/PoliciesTable/PoliciesTableConfig.tsx

```diff
@@ -175,6 +174,27 @@ const generateTableHeaders = (
                   </ReactTooltip>
                 </>
               )}
+              {viewingTeamPolicies && !cellProps.row.original.team_id && (
+                <>
+                  <span
```

Comment:
Made a component for this in [this pr](https://github.com/fleetdm/fleet/pull/18596)